nix-tools-iohk-module.nix

```diff
@@ -26,69 +27,85 @@ let
     # run on the target host.
     inherit (config.hsPkgs.remote-iserv.components.exes) remote-iserv;
     # we need to use openssl.bin here, because the .dll's are in the .bin expression.
-    extra-test-libs = [ pkgs.rocksdb pkgs.openssl.bin pkgs.libffi ];
-  } // {
-    # we can perform testing of cross compiled test-suites by using wine.
-    # Therfore let's enable doCrossCheck here!
-    doCrossCheck = pkgs.stdenv.hostPlatform.isWindows;
   };
-in {
-  packages = {
-    # This needs true, otherwise we miss most of the interesting
-    # modules.
-    ghci.flags.ghci = true;
-    # I hope we can apply this globally.
-    ghc.flags.ghci = true;
+  windowsArgs = withTHArgs // { extra-test-libs = [ pkgs.rocksdb pkgs.openssl.bin pkgs.libffi ]; };
+  linuxArgs = withTHArgs // { extra-test-libs = []; };
+in lib.mkMerge [
+  {
+    packages = {
+      # This needs true, otherwise we miss most of the interesting
+      # modules.
+      ghci.flags.ghci = true;
+      # I hope we can apply this globally.
+      ghc.flags.ghci = true;
 
-    # this needs to be true to expose module
-    #  Message.Remote
-    # as needed by libiserv.
-    libiserv.flags.network = true;
+      # this needs to be true to expose module
+      #  Message.Remote
+      # as needed by libiserv.
+      libiserv.flags.network = true;
 
-    # libiserv has a bit too restrictive boundaries.
-    # as such it won't build with newer network libraries.
-    # to avoid that we use doExactConfig, which forces cabal
-    # to forgoe its solver and just take the libraries it's
-    # provided with.
-    ghci.components.library.doExactConfig = true;
-    libiserv.components.library.doExactConfig = true;
-    # same for iserv-proxy
-    iserv-proxy.components.exes.iserv-proxy.doExactConfig = true;
-    remote-iserv.components.exes.remote-iserv.doExactConfig = true;
-    remote-iserv.postInstall = with pkgs.stdenv; lib.optionalString hostPlatform.isWindows ''
-      cp ${pkgs.libffi}/bin/*.dll $out/bin/
-    '';
+      # libiserv has a bit too restrictive boundaries.
+      # as such it won't build with newer network libraries.
+      # to avoid that we use doExactConfig, which forces cabal
+      # to forgoe its solver and just take the libraries it's
+      # provided with.
+      ghci.components.library.doExactConfig = true;
+      libiserv.components.library.doExactConfig = true;
+      # same for iserv-proxy
+      iserv-proxy.components.exes.iserv-proxy.doExactConfig = true;
+      remote-iserv.components.exes.remote-iserv.doExactConfig = true;
+      remote-iserv.postInstall = with pkgs.stdenv; lib.optionalString hostPlatform.isWindows ''
+        cp ${pkgs.libffi}/bin/*.dll $out/bin/
+      '';
 
-    # clock hasn't had a release since 2016(!) that is for three(3) years
-    # now.
```

Comment:
This reminds me, this comment isn't correct anymore. As https://github.com/corsis/clock/issues/55 has been closed a week ago with the release of clock-0.8